data/data/agent/systemd/units/agent-copy-iso-files.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Copy the additional files stored in the agent ISO during the boot
+Wants=network-online.target
+After=network-online.target
+
```

Comment:
I think there should be a Before on agent-tui.path.
https://github.com/openshift/installer/blob/master/data/data/agent/systemd/units/agent-tui.path

Reply:
Not sure I got the rationale. The synchronization point will be in any case set on the presence of `/usr/local/bin/agent-tui`, regardless of the execution ordering of the service - it's not an explicit dependency

Reply:
I was worried that it wouldn't trigger the path dependency in the right time. But it sounds like it ought to work.